net/test/testdata/aci_cfg.yml

```diff
@@ -0,0 +1,17 @@
+# This is the configuration for baremetal used for testing aci installer
+# XXX: for now we have one setup so hardcoding the details, we need to
+# revisit this when we add more aci test setups
+
+CONNECTION_INFO:
+      contiv-aci-server4:
+        control: eth3
+        data: eth7
+      contiv-aci-server5:
+        control: eth3
+        data: eth7
+APIC_URL: "https://contiv-aci-ifc1:443"
+APIC_USERNAME: "admin"
+APIC_PASSWORD: "ins3965!"
```

Comment:
not a big deal @mapuri 
I will merge the changes... so we can get the sanity going on.